include/swift/Option/FrontendOptions.td

```diff
@@ -497,6 +497,9 @@ def enable_class_resilience : Flag<["-"], "enable-class-resilience">,
 def enable_resilience_bypass : Flag<["-"], "enable-resilience-bypass">,
    HelpText<"Completely bypass resilience when accessing types in resilient frameworks">;
 
+def read_type_info_path_EQ : Joined<["-"], "read-type-info-path=">,
+  HelpText<"Read legacy type layout from the given path">;
```

Comment:
It was just so that when testing I can write `-Xfrontend -read-type-info-path=foo` instead of `-Xfrontend -read-type-info-path -Xfrontend foo`. But I think this flag is only temporary, since we'll want to use some other mechanism to find these file(s) eventually, correct?

Reply:
Yeah, probably we'll just look in the resource directory. The -Xfrontend motivation is reasonable but I figure you're going to be using the flag mostly in tests, right?